BlazorAppTest/Pages/HxCalendar_Issue597_Test.razor

```diff
@@ -0,0 +1,49 @@
+﻿@page "/HxCalendar_Issue597_Test"
+@using BlazorAppTest.Pages.HxCalendarComponents;
+@inject TimeProvider SystemTimeProvider
+<h1>HxCalendar</h1>
+<fieldset class="m-2 p-2">
+    <label>From DI: @(SystemTimeProvider.LocalTimeZone.Id)</label>
+    <div>
+        <HxCalendar />
+    </div>
+</fieldset>
+
+<fieldset class="m-2 p-2">
+    <label>From Settings: @(AppCalendarSettings.AmericanSamoa.TimeProvider.LocalTimeZone.Id)</label>
+    <div><HxCalendar Settings="AppCalendarSettings.AmericanSamoa" /></div>
+</fieldset>
+
+<fieldset class="m-2 p-2">
+    <label>From Settings: @(AppCalendarSettings.Samoa.TimeProvider.LocalTimeZone.Id)</label>
+    <div><HxCalendar Settings="AppCalendarSettings.Samoa" /></div>
+</fieldset>
+
+<fieldset class="m-2 p-2">
+    <label>From Settings: @(AppCalendarSettings.timeProviderUtc.LocalTimeZone.Id)</label>
+    <div><HxCalendar TimeProvider="AppCalendarSettings.timeProviderUtc" /></div>
+</fieldset>
+
+@code {
+
+    public class AppCalendarSettings
+    {
+        public static TimeProvider timeProviderUtc = new ZonedTimeProvider(TimeZoneInfo.Utc);
+        public static TimeProvider timeProviderUtc11 = new ZonedTimeProvider(TimeZoneInfo.FindSystemTimeZoneById("UTC-11"));
+        public static TimeProvider timeProviderUtc13 = new ZonedTimeProvider(TimeZoneInfo.FindSystemTimeZoneById("Samoa Standard Time"));
+        public static CalendarSettings AmericanSamoa => new()
+            {
+                TimeProvider = timeProviderUtc11,
+            };
+        public static CalendarSettings Samoa => new()
+            {
+                TimeProvider = timeProviderUtc13,
+            };
```

Comment:
Please adjust according to coding-standards:
* `public` members vs `camelCase` => change the 11 & 13 providers to `private` and move the UTC provider outside the `AppCalendarSettings` class (to the test-component itself)
* indentation of the object initializers